student/src/test/resources/test-context.xml

```diff
@@ -56,7 +56,7 @@
 	<!-- For ResponseRepositoryTest -->	
 	<beans:bean id="iResponseRepository" class="tds.student.sql.repository.ResponseRepository"/>
 	<!-- For PrintServiceTest -->
-	<beans:bean id="printService" class="tds.student.services.PrintService"/>
+	<beans:bean id="printService" class="tds.student.services.PrintServiceImpl"/>

```

Comment:
Is this even needed?  Proctor is using a mix of XML and annotation based injection but it seems to me looking at the PrintServiceImpl class that the bean definition isn't needed?